MiniTwitch.Irc/Models/Privmsg.cs

```diff
@@ -222,6 +247,16 @@ internal Privmsg(ref IrcMessage message, IrcClient? source = null)
                     sub = TagHelper.GetBool(tagValue);
                     break;
 
+                //animation-id
+                case (int)Tags.AnimationId:
+                    animation = tagValue.Sum() switch
+                    {
+                        653 => AnimationId.Simmer,
+                        1540 => AnimationId.RainbowEclipse,
+                        _ => AnimationId.UNKNOWN,
```

Comment:
So if Twitch introduces a new animation, consumers of this library will have to wait for updates before they can read it? Is this data stored anywhere else for them to fall back to if required?

I almost wonder if it would be nicer from a consumer perspective to expose this as the value and let them deal with its contents - which I'm sure you could find a minimal allocation way to do (e.g. interning if the input is sane)

Reply:
good point